### DISCUSSION TITLE
Copilot chat network whitelist

### DISCUSSION THREAD

**Discussion by melandz** [community/community#77913] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello,

We have purchased the GitHub Copilot Business license, but GitHub Copilot Chat is restricted due to an error. The main issue is related to our company's internal network isolation. Could you please assist us in understanding how to set up the network whitelist? Thank you.

We have already set up the following firewall rules:

https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-firewall-settings-for-github-copilot

Domain and/or URL Purpose
https://github.com/login/* Authentication
https://api.github.com/user User Management
https://api.github.com/copilot_internal/* User Management
https://copilot-telemetry.githubusercontent.com/telemetry Telemetry
https://default.exp-tas.com/ Telemetry
https://copilot-proxy.githubusercontent.com/ API service for Copilot suggestions
https://origin-tracker.githubusercontent.com/ API service for Copilot suggestions
https://*.githubcopilot.com API service for Copilot suggestions

**Comment by Akash1134** [comment]
Hi @melandz ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). This will ensure data is not shared publicly including any network terminals, URLs, and endpoints. Please open a ticket with them :)

**Reply by melandz** [reply]
Thank you for your reply.

**Reply by melandz** [reply, marked as answer]
After our testing, do not use the built-in Windows proxy. Instead, using the proxy through the original Internet Explorer interface functions correctly. We speculate that this issue may be related to the Windows SDK version used by Copilot Chat.
